hs_geographic_feature_resource/models.py

```diff
@@ -140,21 +322,39 @@ def has_all_required_elements(self):
         return True
 
     def get_required_missing_elements(self):  # show missing required meta
-        missing_required_elements = super(GeographicFeatureMetaData, self).\
+        missing_required_elements = super(GeographicFeatureMetaDataMixin, self). \
             get_required_missing_elements()
```

Comment:
@pkdash Same question as above, does ```get_required_missing_elements()``` exist in the super class of the Mixin class?